R/read_extract_cmh.R

```diff
@@ -0,0 +1,55 @@
+#' Read CMH extract
+#'
+#' @param year Year of BOXI extract
+#'
+#' @return csv data file for Acute
+#' @export
+#'
+read_extract_cmh <- function(year) {
+  extract_cmh_path <- get_boxi_extract_path(year = year, type = "CMH")
+
+  # Read BOXI extract
+  extract_cmh <- readr::read_csv(extract_cmh_path,
+    col_types = cols_only(
+      "UPI Number [C]" = col_character(),
+      "Patient DoB Date [C]" = col_date(format = "%Y/%m/%d %T"),
+      "Gender" = col_double(),
+      "Patient Postcode [C]" = col_character(),
+      "NHS Board of Residence Code 9" = col_character(),
+      "Patient HSCP Code - current" = col_character(),
+      "Practice Code" = col_character(),
+      "Treatment NHS Board Code 9" = col_character(),
+      "Contact Date" = col_date(format = "%Y/%m/%d %T"),
+      "Contact Start Time" = col_time(format = "%T"),
+      "Duration of Contact" = col_time(format = "%M"),
```

Comment:
```suggestion
      "Duration of Contact" = col_integer(),
```
Adding this as I think `col_time()` was trying to parse this as a time of day rather than a duration of minutes

Reply:
Amazing! thanks for looking into this! Just tested this and also working fine for me! :)